main.go

```diff
@@ -563,3 +568,56 @@ func deployCloudComponents(plan types.Plan) error {
 
 	return nil
 }
+
+func featureEnabled(features []string, secretFeatures []string) bool {
+	for _, feature := range features {
+		for _, secretFeature := range secretFeatures {
+			if feature == secretFeature {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func filterFeatures(plan types.Plan) types.Plan {
+	plan.Features = append(plan.Features, types.DefaultFeature)
+
+	if (plan.S3 != types.S3{}) {
```

Comment:
S3 should not be optional in any case, what is this doing?

Reply:
Forgot to remove that from the code, will do 👍

Reply:
Removed in latest commit ✔️